src/screens/Dashboard/Dashboard.tsx

```diff
@@ -5,7 +5,7 @@ import styles from '@screens/Dashboard/styles';
 function Dashboard() {
   return (
     <SafeAreaView>
```

Comment:
why are you using safeAreaView everywhere instead of this wrap main comp inside safeAreaView